spec/forms/signin/mobile_token_form_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe Signin::MobileTokenForm, type: :model do
+  subject(:form) { described_class.new(attrs) }
+  let(:attrs) { { email: user.email } }
+  let(:user) { create(:user) }
+
+  context 'validations' do
+    it { should validate_presence_of(:email) }
+
+    describe 'user existence' do
+      before { attrs.merge!(email: Faker::Internet.email) }
```

Comment:
```suggestion
      let(:attrs) { { email: Faker::Internet.email } }
```
Does the same right?

Reply:
Yes! 2f7fa38ae843a061dae32cb3634c374892fcb37a